pkg/server/controller.go

```diff
@@ -219,6 +219,16 @@ func (ctrl *Controller) serverMux() (http.Handler, error) {
 		apiRouter.RegisterAdhocHandlers()
 	}
 
+	// FIXME: Honor ctrl.config.RemoteRead.Enabled.
+	authorizer := authz.NewAuthorizer(ctrl.log, httputils.NewDefaultHelper(ctrl.log))
+	appsRouter := apiRouter.PathPrefix("/apps").Subrouter()
+	appsRouter.Use(ctrl.authMiddleware(nil))
```

Comment:
`apiRouter` already has auth middleware, I'm not sure we need registering it one more time for a sub-router